<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDisplayDialog</class>
 <widget class="QDialog" name="LogDisplayDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AspeQt Log View</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QTextEdit" name="textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Filter log by:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="listByDisk">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>ALL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Disk 15</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>LogDisplayDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>LogDisplayDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listByDisk</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>LogDisplayDialog</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
